<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiDataVaultDialog</class>
 <widget class="QDialog" name="UiDataVaultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>345</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayoutDataVaultDialog">
   <property name="spacing">
    <number>12</number>
   </property>
   <item>
    <widget class="QLabel" name="labelVaultName">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data Vault Name</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelVaultNameDescription">
     <property name="text">
      <string>Please choose a name for the new data vault.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="buddy">
      <cstring>lineEditVaultName</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEditVaultName"/>
   </item>
   <item>
    <widget class="QLabel" name="labelVaultPassword">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Data Vault Password</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelVaultPasswordDescriptionTop">
     <property name="text">
      <string>Choose an secure password possible with at least 6 letters, numbers and special characters.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPasswordBox">
     <item>
      <spacer name="horizontalSpacerPassword">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>60</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutPasswordBox">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutPassword">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="labelPassword">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Password:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
           <property name="buddy">
            <cstring>lineEditPassword</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPassword"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;
&lt;body&gt;
    &lt;h6&gt;Password Rules&lt;/h6&gt;
    &lt;ul&gt;
        &lt;li&gt;At least one lower and upper case letter: a-z A-Z öäü ÖÄÜ&lt;/li&gt;
        &lt;li&gt;At least one digit: 0-9&lt;/li&gt;
        &lt;li&gt;At least one of special character: !&amp;quot;§$%&amp;/()=?&amp;#180;&amp;#96;{}[]\ß&amp;#64;€~’*'+#-_.:,;µ&amp;ouml;&amp;auml;&amp;uuml;&amp;Ouml;&amp;Auml;&amp;Uuml;&amp;lt;|&amp;gt;&lt;/li&gt;
        &lt;li&gt;Minimum six in length 6 (with the anchors)&lt;/li&gt;
    &lt;/ul&gt;
&lt;/body&gt;
&lt;/html&gt;
</string>
           </property>
           <property name="pixmap">
            <pixmap resource="../../res/olbaflinx.qrc">:/general/question</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutPasswordConfirm">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QLabel" name="labelPasswordConfirm">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Password confirm:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="indent">
            <number>5</number>
           </property>
           <property name="buddy">
            <cstring>lineEditPasswordConfirm</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPasswordConfirm"/>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;
&lt;body&gt;
    &lt;h6&gt;Password Rules&lt;/h6&gt;
    &lt;ul&gt;
        &lt;li&gt;At least one lower and upper case letter: a-z A-Z öäü ÖÄÜ&lt;/li&gt;
        &lt;li&gt;At least one digit: 0-9&lt;/li&gt;
        &lt;li&gt;At least one of special character: !&amp;quot;§$%&amp;/()=?&amp;#180;&amp;#96;{}[]\ß&amp;#64;€~’*'+#-_.:,;µ&amp;ouml;&amp;auml;&amp;uuml;&amp;Ouml;&amp;Auml;&amp;Uuml;&amp;lt;|&amp;gt;&lt;/li&gt;
        &lt;li&gt;Minimum six in length 6 (with the anchors)&lt;/li&gt;
    &lt;/ul&gt;
&lt;/body&gt;
&lt;/html&gt;
</string>
           </property>
           <property name="pixmap">
            <pixmap resource="../../res/olbaflinx.qrc">:/general/question</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="labelVaultPasswordDescriptionBottom">
     <property name="text">
      <string>The password is known only to you and cannot be reset. If you forget the password, you will not be able to access the data vault.</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDataVaultDialogButtons">
     <item>
      <spacer name="horizontalSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset theme="cancel">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOk">
       <property name="text">
        <string>&amp;Ok</string>
       </property>
       <property name="icon">
        <iconset theme="checkmark">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditVaultName</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditPasswordConfirm</tabstop>
  <tabstop>pushButtonOk</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../../res/olbaflinx.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>UiDataVaultDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonOk</sender>
   <signal>clicked()</signal>
   <receiver>UiDataVaultDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>326</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
